src/ROUTES.js

```diff
@@ -56,6 +56,8 @@ export default {
     REPORT,
     REPORT_WITH_ID: 'r/:reportID',
     getReportRoute: reportID => `r/${reportID}`,
+    SELECT_YEAR: 'select-year',
+    getYearSelectionRoute: (minYear, maxYear, currYear, backTo) => `select-year?min=${minYear}&max=${maxYear}&year=${currYear}&backTo=${backTo}`,
```

Comment:
Also super NAB - i feel like `backTo` could be slightly clearer, so maybe we can explain in a comment or change to `originatingRoute` or something like that? If y'all agree it's not immediately clear